debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SourceInfo.java

```diff
@@ -501,19 +505,42 @@ public void setOffset(Map<String, ?> sourceOffset) {
             this.restartEventsToSkip = longOffsetValue(sourceOffset, EVENTS_TO_SKIP_OFFSET_KEY);
             nextSnapshot = booleanOffsetValue(sourceOffset, SNAPSHOT_KEY);
             lastSnapshot = nextSnapshot;
-            this.databaseWhitelist = (String) sourceOffset.get(DATABASE_WHITELIST_KEY);
-            this.databaseBlacklist = (String) sourceOffset.get(DATABASE_BLACKLIST_KEY);
-            this.tableWhitelist = (String) sourceOffset.get(TABLE_WHITELIST_KEY);
-            this.tableBlacklist = (String) sourceOffset.get(TABLE_BLACKLIST_KEY);
+            if (sourceOffset.containsKey(DATABASE_INCLUDE_LIST_KEY)) {
```

Comment:
How about using `getOrDefault` method to simplify the code?

Reply:
I think this code is slightly faster or equal than the getOrDefault version. I would have used getOrDefault if I could've been able to pass a lambda as default but in the getOrDefault version I would need to do 2 complete value lookups. here it's a key existence check and fetching 1 entry from the map. it might be the same performance if both keys are in the same bucket but worse when in 2 different buckets.

Reply:
ok